<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1187</width>
    <height>759</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="opaqueResize">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget">
       <layout class="QVBoxLayout" name="leftPart">
        <item>
         <widget class="QWidget" name="widget" native="true">
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QSplitter" name="splitter_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <widget class="QWidget" name="verticalLayoutWidget_3">
              <layout class="QVBoxLayout" name="leftTopPart">
               <item>
                <widget class="QGroupBox" name="gb_calibmode">
                 <property name="title">
                  <string>Calibration Mode</string>
                 </property>
                 <layout class="QVBoxLayout" name="verticalLayout_12">
                  <item>
                   <layout class="QVBoxLayout" name="layout_selectCalibMode"/>
                  </item>
                 </layout>
                </widget>
               </item>
               <item>
                <widget class="QGroupBox" name="gb_realtimeCalib">
                 <property name="title">
                  <string>Realtime calibration</string>
                 </property>
                 <layout class="QVBoxLayout" name="verticalLayout_10">
                  <item>
                   <widget class="QLabel" name="label">
                    <property name="text">
                     <string>In development</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
               <item>
                <widget class="QGroupBox" name="gb_filemodeCalib">
                 <property name="title">
                  <string>Filemode Calibration</string>
                 </property>
                 <layout class="QVBoxLayout" name="verticalLayout_9">
                  <item>
                   <layout class="QHBoxLayout" name="layout_filemodeCalib"/>
                  </item>
                 </layout>
                </widget>
               </item>
              </layout>
             </widget>
             <widget class="QWidget" name="verticalLayoutWidget_4">
              <layout class="QVBoxLayout" name="leftBottomPart">
               <item>
                <widget class="QWidget" name="widget_2" native="true">
                 <layout class="QVBoxLayout" name="verticalLayout_4">
                  <item>
                   <widget class="QSplitter" name="splitter_3">
                    <property name="minimumSize">
                     <size>
                      <width>0</width>
                      <height>400</height>
                     </size>
                    </property>
                    <property name="orientation">
                     <enum>Qt::Vertical</enum>
                    </property>
                    <property name="opaqueResize">
                     <bool>true</bool>
                    </property>
                    <property name="childrenCollapsible">
                     <bool>true</bool>
                    </property>
                    <widget class="QWidget" name="horizontalLayoutWidget">
                     <layout class="QHBoxLayout" name="layout_infoAndControl">
                      <item>
                       <widget class="QGroupBox" name="gb_checkboardInfo">
                        <property name="title">
                         <string>Checkboard information</string>
                        </property>
                        <layout class="QVBoxLayout" name="verticalLayout_11">
                         <item>
                          <layout class="QVBoxLayout" name="layout_checkboardInfo"/>
                         </item>
                        </layout>
                       </widget>
                      </item>
                      <item>
                       <widget class="QGroupBox" name="gb_calibControl">
                        <property name="title">
                         <string>Calibration Control</string>
                        </property>
                        <property name="alignment">
                         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                        </property>
                        <property name="flat">
                         <bool>false</bool>
                        </property>
                        <layout class="QVBoxLayout" name="verticalLayout_6">
                         <item>
                          <layout class="QVBoxLayout" name="layout_calibControl"/>
                         </item>
                        </layout>
                       </widget>
                      </item>
                     </layout>
                    </widget>
                    <widget class="QWidget" name="horizontalLayoutWidget_2">
                     <layout class="QHBoxLayout" name="layout_statusAndDepthMap">
                      <item>
                       <widget class="QGroupBox" name="gb_status">
                        <property name="title">
                         <string>Status</string>
                        </property>
                        <layout class="QVBoxLayout" name="verticalLayout_15">
                         <item>
                          <layout class="QVBoxLayout" name="layout_status"/>
                         </item>
                        </layout>
                       </widget>
                      </item>
                      <item>
                       <widget class="QGroupBox" name="gb_depthMaps">
                        <property name="minimumSize">
                         <size>
                          <width>0</width>
                          <height>225</height>
                         </size>
                        </property>
                        <property name="title">
                         <string>Depth Maps</string>
                        </property>
                        <layout class="QVBoxLayout" name="verticalLayout_13">
                         <item>
                          <layout class="QVBoxLayout" name="layout_depthMap"/>
                         </item>
                        </layout>
                       </widget>
                      </item>
                     </layout>
                    </widget>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
               <item>
                <layout class="QFormLayout" name="formLayout">
                 <item row="0" column="0">
                  <widget class="QLabel" name="label_numberOfImgs">
                   <property name="text">
                    <string>Number of pair image or images:</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="1">
                  <widget class="QLabel" name="label_count">
                   <property name="minimumSize">
                    <size>
                     <width>75</width>
                     <height>15</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>75</width>
                     <height>23</height>
                    </size>
                   </property>
                   <property name="layoutDirection">
                    <enum>Qt::LeftToRight</enum>
                   </property>
                   <property name="frameShape">
                    <enum>QFrame::Box</enum>
                   </property>
                   <property name="text">
                    <string/>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="verticalLayoutWidget_2">
       <layout class="QVBoxLayout" name="rightPart">
        <item>
         <widget class="QWidget" name="widget_3" native="true">
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <widget class="QLabel" name="label_inputImg">
               <property name="text">
                <string>&lt; Input Image &gt;</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QSplitter" name="splitter_4">
             <property name="minimumSize">
              <size>
               <width>740</width>
               <height>0</height>
              </size>
             </property>
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <widget class="QWidget" name="horizontalLayoutWidget_3">
              <layout class="QHBoxLayout" name="rightTopPart">
               <item>
                <widget class="QGroupBox" name="gb_inputCamera1">
                 <property name="title">
                  <string>camera1</string>
                 </property>
                 <layout class="QVBoxLayout" name="verticalLayout_2">
                  <item>
                   <layout class="QVBoxLayout" name="layout_leftInput"/>
                  </item>
                 </layout>
                </widget>
               </item>
               <item>
                <widget class="QGroupBox" name="gb_inputCamera2">
                 <property name="title">
                  <string>camera2</string>
                 </property>
                 <layout class="QVBoxLayout" name="verticalLayout_5">
                  <item>
                   <layout class="QVBoxLayout" name="layout_rightInput"/>
                  </item>
                 </layout>
                </widget>
               </item>
              </layout>
             </widget>
             <widget class="QWidget" name="horizontalLayoutWidget_4">
              <layout class="QHBoxLayout" name="rightBottomPart">
               <item>
                <widget class="QGroupBox" name="gb_outputCamera1">
                 <property name="title">
                  <string>camera1</string>
                 </property>
                 <layout class="QVBoxLayout" name="verticalLayout_8">
                  <item>
                   <layout class="QVBoxLayout" name="layout_leftOutput"/>
                  </item>
                 </layout>
                </widget>
               </item>
               <item>
                <widget class="QGroupBox" name="gb_outputCamera2">
                 <property name="title">
                  <string>camera2</string>
                 </property>
                 <layout class="QVBoxLayout" name="verticalLayout_7">
                  <item>
                   <layout class="QVBoxLayout" name="layout_rightOutput"/>
                  </item>
                 </layout>
                </widget>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QLabel" name="label_outputImg">
               <property name="text">
                <string>&lt; Output Image &gt;</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1187</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
